modules/subscriptions/views.php

```diff
@@ -150,6 +150,12 @@ static function render_widget_status_messages( $instance ) {
 							__( 'Manage your email preferences.', 'jetpack' )
 						); ?></p>
 					<?php break;
+				case 'many_pending_subs' : ?>
+					<p class="error"><?php printf( __( 'You already have several pending email subscriptions. <br /> Approve or delete a few subscriptions at <a href="%1$s" title="%2$s" target="_blank">subscribe.wordpress.com</a> before continuing.', 'jetpack' ),
```

Comment:
I don't know if `several` is the right word here, since this can happen with just a few pending subs. Should we remove that word?

Reply:
`several` means `more than two but not many.` (I checked on the Oxford dictionary :D) Basically, its meaning is similar to "a few". 

Originally, I used "a few" too and then I followed the same message in WP.com side
https://github.com/Automattic/jetpack/blob/8.0/modules/subscriptions/views.php#L177

Reply:
I'll let a native chime in on this :)

cc @kraftbj